ddtrace/tracer/option.go

```diff
@@ -220,6 +224,17 @@ func newConfig(opts ...StartOption) *config {
 	c.profilerEndpoints = internal.BoolEnv(traceprof.EndpointEnvVar, true)
 	c.profilerHotspots = internal.BoolEnv(traceprof.CodeHotspotsEnvVar, true)
 
+	schemaVersionStr := os.Getenv(namingschema.SpanAttributeSchemaEnvVar)
+	if v, ok := namingschema.ParseVersion(schemaVersionStr); ok {
+		namingschema.SetVersion(v)
+		c.spanAttributeSchemaVersion = int(v)
+	} else {
+		defaultSchema := namingschema.SchemaV0
+		namingschema.SetVersion(defaultSchema)
+		c.spanAttributeSchemaVersion = int(defaultSchema)
+		log.Warn("DD_TRACE_SPAN_ATTRIBUTE_SCHEMA=%s is not a valid value, setting to default of v0", schemaVersionStr)
```

Comment:
Nit: Let's handle the defaulting in the `namingschema` package, pkg tracer doesn't really have to know how to default the schema version.
```suggestion
		namingschema.SetDefaultVersion()
		v := namingschema.GetVersion()
		c.spanAttributeSchemaVersion = v
		log.Warn("DD_TRACE_SPAN_ATTRIBUTE_SCHEMA=%s is not a valid value, defaulting to %d", v)
```